public/chart.js

```diff
@@ -0,0 +1,57 @@
+//Declare variables
+let positive = 0
+let negative = 0
+let neutral = 0
+
+//Creates a new chart every call
+function createChart(...data) {
```

Comment:
Can you refactor your code to make this a pure function that returns a new `$chart` element? You can append the return value. Try to avoid side-effects in your functions.